### ISSUE TITLE
Error: test show nothing

### ISSUE THREAD

**Issue by TriBall3** [HaojHuang/Equivariant-Transporter-Net#2]
Hi! I tested the trained `checkpoints` file, and the following appeared in the terminal. It seems that there is no error. May I ask if this is testing? Where is the `.pkl` file?
> (etn) fty98@debian-rtx8000:~/lab/fty/Equivariant-Transporter-Net$ python gripper_test.py --equ --lite --angle_lite --task=block-insertion --n_demos 1 --n_steps 10002022-04-01 12:15:20.677732: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: :/usr/local/cuda/lib64
> 2022-04-01 12:15:20.677804: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
> pybullet build time: Mar 26 2022 03:00:15
> text argument:./raven/assets
> equi agent
> /home/fty98/.conda/envs/etn/lib/python3.7/site-packages/e2cnn/nn/modules/r2_conv/basisexpansion_singleblock.py:80: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (Triggered internally at  /opt/conda/conda-bld/pytorch_1616554786529/work/aten/src/ATen/native/IndexingUtils.h:30.)
>   full_mask[mask] = norms.to(torch.uint8)
> Loading pre-trained model at 1000 iterations.
> ./checkpoints_equ/block-insertion-1-0/transport-ckpt2-1000.pt
> int args: [

**Comment by HaojHuang** [comment]
Did it show the rewards of each episode when you run gripper_test.py? The pkl file should be stored in ./test_equi folder.
